src/ui/prompts/listPrompts/variationsListPrompt.ts

```diff
@@ -16,19 +19,74 @@ export class VariationListOptions extends ListOptionsPrompt<CreateVariationParam
             ExitPrompt
         ]
     }
+
+    featureVariables: Variable[] = []
+
+    getVariationListPrompt = (existingVariations?: Variation[], existingVariables?: Variable[]) => {
+        this.featureVariables = existingVariables || []
+
+        return {
+            name: 'variations', 
+            value: 'variations', 
+            message: 'Manage variations',
+            type: 'listOptions',
+            previousReponseFields: ['variables'], // if variables were just created / updated, use the new variables
+            listOptionsPrompt: (previousResponses?: Record<string, any>) => {
+                const newVariables = previousResponses?.variables
+                this.featureVariables = newVariables || []
+
+                return this.prompt(existingVariations?.map((variation) => ({
+                    name: variation.name || variation.key,
+                    value: variation
+                })))
+            }
+        }
+    }
+
     async promptAddItem<CreateVariationParams>(): Promise<ListOption<CreateVariationParams>> {
-        // TODO: Implement add item
+        const variation = await inquirer.prompt(staticCreateVariationPrompts)
+        variation.variables = await inquirer.prompt(getVariationVariablesPrompts(this.featureVariables))
+
         return {
-            name: 'Placeholder',
-            value: {} as CreateVariationParams
+            name: variation.name,
+            value: variation as CreateVariationParams
         }
     }
 
     async promptEditItem<CreateVariationParams>(
         list: ListOption<CreateVariationParams>[]
-    ): Promise<ListOption<CreateVariationParams>> {
-        // TODO: Implement edit item
-        return Promise.resolve(list[0])
+    ): Promise<void> {
+        if (list.length === 0) {
+            this.writer.warningMessage('No variations to edit')
+            return
+        }
+        const response = await inquirer.prompt([{
+            name: 'variationToEdit',
+            message: 'Which variation would you like to edit?',
+            type: 'list',
+            choices: list
+        }])
+        const index = list.findIndex((listItem) => listItem.name === response.variationToEdit.name)
+
+        // Have a default for each of the prompts that correspond to the previous value of the variable
+        const filledOutPrompts = staticCreateVariationPrompts.map((prompt) => ({
+            ...prompt,
+            default: response.variationToEdit[prompt.name]
```

Comment:
isn't `response.variationToEdit` an object in the form
```
{
	name: "string",
	value: variation
}
```

Reply:
hm it seems like response.variationToEdit just has the value field? the default values are working anyway

Reply:
ah yeah, the answer is just the value field, forgot about that